includes/templates/class-amp-post-template.php

```diff
@@ -80,14 +80,50 @@ class AMP_Post_Template {
 	 */
 	public $post;
 
+	/**
+	 * Custom template root.
+	 *
+	 * @since 0.7
+	 * @var bool
+	 */
+	public $custom_template_dir;
+
+	/**
+	 * If template system is currently active.
+	 *
+	 * @since 0.7
+	 * @var bool
+	 */
+	public $active;
```

Comment:
I feel the term "active" might be too ambiguous here. This is what the callback property within the theme support addition controls, and is used when we do have a custom template included but want to control if it should be AMP or default WP templates (since the custom will try to follow WP template hierarchy and we may not always want it to).